apps/server/test/lexical/server/provider/handlers/go_to_definition_test.exs

```diff
@@ -22,13 +22,16 @@ defmodule Lexical.Server.Provider.Handlers.GoToDefinitionTest do
         {DynamicSupervisor, name: Server.Project.Supervisor.dynamic_supervisor_name()}
       )
 
+    {:ok, _} = start_supervised(Lexical.RemoteControl.ProjectNodeSupervisor)
     {:ok, _} = start_supervised({Server.Project.Supervisor, project})
 
     Dispatch.register(project, [project_compiled()])
     RemoteControl.Api.schedule_compile(project, true)
 
     assert_receive project_compiled(), 5000
 
+    on_exit(fn -> RemoteControl.stop(project) end)
```

Comment:
this is an indication that bidirectional monitoring isn't working properly. You should not have to stop anything manually that was started via `start_supervised`.